index.js

```diff
@@ -63,8 +63,9 @@ function plugin(opts) {
 
         debug('adding %s to includes as `%s`', resolvedFilename, name);
 
-        file[name] = included[filename].contents;
-
+        file[name] = included[filename];
+        file[name].contents = included[filename].contents;
```

Comment:
@RobLoach we need the contents key now that we are including the YAML frontmatter keys in the object. So file[name] is no longer contents, but an object containing contents and any frontmatter.